src/content_sao.cpp

```diff
@@ -477,7 +483,9 @@ void LuaEntitySAO::step(float dtime, bool send_recommended)
 		move_d += m_last_sent_move_precision;
 		float vel_d = m_velocity.getDistanceFrom(m_last_sent_velocity);
 		if (move_d > minchange || vel_d > minchange ||
-				std::fabs(m_yaw - m_last_sent_yaw) > 1.0) {
+				std::fabs(m_yaw - m_last_sent_yaw) ||
+				std::fabs(m_pitch - m_last_sent_pitch) ||
```

Comment:
`> 1.0`

Reply:
`1.0f` is better to define a 'float' instead of a 'double'